packages/dd-trace/src/plugins/util/ip_extractor.js

```diff
@@ -62,6 +66,21 @@ function extractIp (config, req) {
   return firstPrivateIp || req.socket?.remoteAddress
 }
 
+function isPublicIp (chunk, type) {
+  return !privateIPMatcher.check(chunk, type === 6 ? 'ipv6' : 'ipv4')
```

Comment:
it's called IP, in the calling function
```suggestion
function isPublicIp (ip, type) {
  return !privateIPMatcher.check(ip, type === 6 ? 'ipv6' : 'ipv4')
```